vpdq/cpp/pdq/cpp/common/pdqhashtypes.h

```diff
@@ -12,7 +12,8 @@
 #include <pdq/cpp/common/pdqbasetypes.h>
 #include <pdq/cpp/common/pdqhamming.h>
 
-#include <stdio.h>
+#include <bitset>
+#include <cinttypes>
```

Comment:
```suggestion
#include <cstdint>
```

Replace `<cinttypes>` with `<cstdint>`. It's smaller than <cinttypes> for the fixed-sized types.

Is `<bitset>` used in this file anymore or can it be removed?